package.json

```diff
@@ -9,7 +9,8 @@
   },
   "peerDependencies": {
     "react": "*",
-    "react-native": "*"
+    "react-native": "*",
+    "react-native-webview": "*"
```

Comment:
@TriPSs any particular reason your adding this as a peer dependency instead of a direct dependency?

Reply:
I think this was to not force a specific version to the users and to keep inline with react native, as that one was also in the peerDependencies.